04/cipher_caesar.py

```diff
@@ -0,0 +1,41 @@
+diacritics = ['á', 'é', 'í', 'ó', 'ú', 'ý', 'č', 'ď', 'ě', 'ň', 'ř', 'š', 'ť', 'ž', 'ů']
+ASCII_numbers = [49, 50, 51, 52, 53, 54, 55, 56, 57, 58]
+
+key = input('key ')
+for znak in key:
+    znak_ASCII = ord(znak)
+    if znak_ASCII not in ASCII_numbers:
+        key = int(input('Zadávej jen celá, kladná čísla. Příklad: 2, 5, 15 \nKey: '))
+
+key = int(key)
+
+if key <= 0:
```

Comment:
Musí to byť naozaj len kladné číslo?

Reply:
Jasné, určite by som to nelimitovala, postačí aby to bolo celé číslo a ten program sa podǎl toho zachová :slightly_smiling_face: